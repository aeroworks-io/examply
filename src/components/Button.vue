<template>
  <v-btn>Button</v-btn>
</template>

<script lang="ts">
import { Component, Vue } from 'vue-property-decorator';

@Component
export default class Button extends Vue {}
</script>
