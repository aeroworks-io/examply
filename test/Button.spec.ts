import { mount } from '@vue/test-utils';
import Button from '@/components/Button.vue';

describe('Button.vue', () => {
  it('snapshot', () => {
    const wrapper = mount(Button);
    expect(wrapper.html()).toMatchSnapshot();
  });
});
