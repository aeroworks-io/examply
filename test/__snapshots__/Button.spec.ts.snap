// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button.vue snapshot 1`] = `"<button type=\\"button\\" class=\\"v-btn v-btn--contained theme--light v-size--default\\"><span class=\\"v-btn__content\\">Button</span></button>"`;
